2.4/test/run

```diff
@@ -160,7 +160,7 @@ for server in ${WEB_SERVERS[@]}; do
   # it from Docker hub
   s2i_args="--pull-policy=never"
 
-  prepare ${server}
```

Comment:
I think this is explained in
https://github.com/TomasTomecek/s2i-ruby-container/blob/87e9448c67df21660861f4a00551f050067fa425/2.4/test/run#L46